lib/game/doodle_dash.dart

```diff
@@ -125,7 +124,7 @@ class DoodleDash extends FlameGame
     );
 
     // reset the the platforms
-    platformManager = PlatformManager(maxVerticalDistanceToNextPlatform: 350);
+    platformManager = PlatformManager();
```

Comment:
It seems like we don't actually need this to a be an argument. If we add difficulty settings in a hypothetical menu, we will pass in a difficulty setting and use that to choose a level from the Map you added. Which I like better.